### ISSUE TITLE
Possibility to increase simulation speed

### ISSUE THREAD

**Issue by Mcpg** [Technochips/gravity#7]
It'd be really nice to speed up the simulation with keys like + and - or something.

**Comment by Technochips** [comment]
I would have to implement delta times and stuff, but i'm not sure how

**Comment by Mcpg** [comment]
Delta time is actually pretty easy.

`function love.update(dt)`

dt is the delta time.
https://love2d.org/wiki/dt

**Comment by Technochips** [comment]
I knew that. Speeding up the simulation that way would decrease precision, also too many keys are already set

**Comment by Mcpg** [comment]
Why not just make a local variable/field name TimeScale, or something and everything related to move be multiplied by this value? It shouldn't be that hard, as you probably know the entire code.

**Comment by Technochips** [comment]
Yes, that's how precision also changes. If you go too fast, stars might go too far away.

**Comment by Mcpg** [comment]
implement it
